userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -194,22 +195,24 @@ falco::app::run_result falco::app::actions::init_inspectors(falco::app::state& s
 				}
 			}
 
-			// init the plugin, if we registered it into an inspector
+			// init the plugin only if we registered it into an inspector
 			// (in capture mode, this is true for every plugin)
-			if(plugin) {
-				// avoid initializing the same plugin twice in the same
-				// inspector if we're in capture mode
-				if(!s.is_capture_mode() || used_plugins.find(p->name()) == used_plugins.end()) {
-					if(!plugin->init(config->m_init_config, err)) {
-						return run_result::fatal(err);
-					}
-				}
-				if(is_input) {
-					auto gen_check = src_info->inspector->new_generic_filtercheck();
-					src_info->filterchecks->add_filter_check(std::move(gen_check));
+			if(!plugin) {
```

Comment:
Same issue as before in this case.

Reply:
Mh... I didn't move it since I think that this comment is more related to the reason why we are `continue`ing there. My interpretation is: "don't initialize the plugin if it hasn't been registered into an inspector". Maybe I can rephrase the comment this way...

Reply:
Applied suggestion